app/assets/stylesheets/course/forum/posts.scss

```diff
@@ -0,0 +1,5 @@
+.course-forum-posts {
+  &.reply {
+    @include forum-post;
```

Comment:
hmm? No there is a post in reply page. That is the post that you are replying to.